Sources/SKSwiftPMWorkspace/SwiftPMWorkspace.swift

```diff
@@ -188,8 +188,7 @@ public actor SwiftPMWorkspace {
         buildSetup: buildSetup,
         reloadPackageStatusCallback: reloadPackageStatusCallback
       )
-    } catch Error.noManifest(let path) {
-      logger.error("could not find manifest, or not a SwiftPM package: \(path)")
```

Comment:
An info log/similar is probably still useful?

Reply:
I don’t think so